ntbs-service/Pages/Alerts/TransferRequest.cshtml.cs

```diff
@@ -151,6 +172,69 @@ public async Task<JsonResult> OnGetFilteredCaseManagersListByTbServiceCode(strin
                     })
                 });
         }
+    }
+
+    public class TransferRequest

```

Comment:
Feels slightly off defining it here although as mentioned in the overall review comment don't have a better suggestion!